team_test.go

```diff
@@ -797,7 +797,7 @@ func TestTeamAddMemberhip(t *testing.T) {
 	team, _ := clientWithAlias.GetTeamWithAlias("example")
 	newMembership := ol.TeamMembershipUserInput{
 		Role: "user",
-		User: ol.UserIdentifierInput{Id: id1, Email: "kyle@opslevel.com"},
+		User: ol.UserIdentifierInput{Id: &id1, Email: ol.NewString("kyle@opslevel.com")},
```

Comment:
I "think" this should probably actually be using `NewUserIdentifier("kyle@opslevel.com")`

Reply:
Introducing `NewUserIdentifier()` here would make the line read as:
`User: NewUserIdentifier("kyle@opslevel.com"),`
but then we are getting a `UserIdentifier` without an `Id`